mambaSharedFramework/Pantos-Generic Playlist Parsing/Pantos-Generic Tag Validators/MasterPlaylistValidator.swift

```diff
@@ -27,9 +27,9 @@ public protocol MasterPlaylistValidator {
      
      - parameter masterPlaylist: A `MasterPlaylistInterface` to validate
      
-     - returns: An array of `HLSValidationIssue`s. Will be empty if no issues are found.
+     - returns: An array of `ValidationIssue`s. Will be empty if no issues are found.
```

Comment:
`ValidationIssue` is used in this swiftdoc, `PlaylistValidationIssue` is used in the function definition below.